enzyme/benchmarks/ReverseMode/adbench/gmm.h

```diff
@@ -168,8 +168,8 @@ int main(const int argc, const char* argv[]) {
     std::vector<std::string> paths;// = { "1k/gmm_d10_K100.txt" };
 
     getTests(paths, "data/1k", "1k/");
-    getTests(paths, "data/2.5k", "2.5k/");
-    getTests(paths, "data/10k", "10k/");
+    //getTests(paths, "data/2.5k", "2.5k/");
```

Comment:
Why is this commented out?

Reply:
Some of the test files for some benchmarks were commented out before due to runtime. 
I can drop this change here, but I'd suggest that we cut the number of tests that we run in CI
to keep CI time reasonable, and once we fixed all tests make it mandatory, to prevent this from regressing again.

Reply:
This has been replaced by the `BENCH_LARGE` environment variable so I'll resolve this thread.